common/changes/@microsoft/rush/publish-auth_2019-03-19-20-45.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Publish: pass auth token through to npm view when checking if package exists",
+      "packageName": "@microsoft/rush",
+      "type": "none"
```

Comment:
I don't think you want this to be `none`, maybe...

Reply:
Rush publishing is done manually, so it's supposed to be this way.

Reply:
okay rush publishing is done manually, marking it approved instead!